src/github.com/getlantern/flashlight/config/config.go

```diff
@@ -357,19 +359,36 @@ func (cfg Config) doFetchCloudConfig(proxyAddr string) ([]byte, error) {
 	if err != nil {
 		return nil, fmt.Errorf("Unable to initialize HTTP client: %s", err)
 	}
+	country := strings.ToLower(globals.GetCountry())
+	if country != "" {
+		ret, err := cfg.fetchCloudConfigForCountry(client, country)
+		// We could check specifically for a 404, but S3 actually returns a 403 when
+		// a resource is not available.  I thought we'd better lean on the side of
+		// robustness by avoiding hardcoding an S3-ism here, than to try and save an
+		// extra request every now and then.
+		if err == nil {
+			return ret, err
+		} else {
+			log.Debugf("Couldn't fetch cloud config for country '%s'; trying the default one", country)
+		}
+	}
+	return cfg.fetchCloudConfigForCountry(client, "default")
+}
 
-	log.Debugf("Checking for cloud configuration at: %s", cfg.CloudConfig)
-	req, err := http.NewRequest("GET", cfg.CloudConfig, nil)
+func (cfg Config) fetchCloudConfigForCountry(client *http.Client, country string) ([]byte, error) {
+	url := strings.Replace(cfg.CloudConfig, countryPlaceholder, country, 1)
+	log.Debugf("Checking for cloud configuration at: %s", url)
```

Comment:
Just to confirm, in server-mode, the way to use a country-specific cloud config is via the ```-cloudconfig http://s3.amazonaws.com/lantern_config/cloud.$\{COUNTRY\}.yaml.gz``` command line designation. Is that correct? This performs the substitution with whatever ISO country code ``getCountry()`` in ``globals.go`` returns; for our Tokyo servers, that's always expected to return "cn" (case-sensitive) to fetch ``http://s3.amazonaws.com/lantern_config/cloud.cn.yaml.gz``?